.github/workflows/workflow.yaml

```diff
@@ -103,9 +103,11 @@ jobs:
         run: |
           pip install --upgrade build
       - name: Build a binary wheel and a source tarball
+        id: pypiBuild
         run: |
           python -m build --sdist --wheel --outdir dist/
       - name: Publish distribution 📦 to Test PyPI
+        if: steps.pypiBuild.outcome == 'success' 
```

Comment:
Do we also need an if for the next step? I can't remember exactly how GitHub actions does it.

Reply:
It will be only for this step